setup.cfg

```diff
@@ -6,7 +6,7 @@ universal=1
 
 [flake8]
 max-line-length = 120
-exclude = build,examples
+exclude = build,examples,kale/utils/print.py,tests/utils/test_print.py
```

Comment:
The change is no needed, already having # noqa. Better to be local rather than global.